{"name": "test_07_enter_credit_card_info[chromium]", "status": "broken", "statusDetails": {"message": "Exception: Element '#card-number' not found within 5000ms.", "trace": "self = <base.ui_actions.UIActions object at 0x1070a8f50>\nlocator = '#card-number', timeout = 5000\n\n    def wait_for_element(self, locator, timeout=5000):\n        self.log.info(f\"Waiting for element: {locator}\")\n        try:\n            element = self.__page.locator(locator)\n>           element.wait_for(state=\"visible\", timeout=timeout)\n\n../base/base_page.py:16: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../.venv/lib/python3.13/site-packages/playwright/sync_api/_generated.py:17907: in wait_for\n    self._sync(self._impl_obj.wait_for(timeout=timeout, state=state))\n../.venv/lib/python3.13/site-packages/playwright/_impl/_locator.py:685: in wait_for\n    await self._frame.wait_for_selector(\n../.venv/lib/python3.13/site-packages/playwright/_impl/_frame.py:323: in wait_for_selector\n    await self._channel.send(\"waitForSelector\", locals_to_params(locals()))\n../.venv/lib/python3.13/site-packages/playwright/_impl/_connection.py:61: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x10710c590>\ncb = <function Channel.send.<locals>.<lambda> at 0x1074cfc40>\nis_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Locator.wait_for: Timeout 5000ms exceeded.\nE           Call log:\nE             - waiting for locator(\"#card-number\") to be visible\n\n../.venv/lib/python3.13/site-packages/playwright/_impl/_connection.py:528: TimeoutError\n\nThe above exception was the direct cause of the following exception:\n\nself = <test_registration.TestLogin object at 0x1070ad7b0>\n\n    @allure.description(\"Test 07 - Fill in credit card info\")\n    def test_07_enter_credit_card_info(self):\n        # Add credit card info before running\n        credit_card = \"4580160117972457\"\n        year_exp = \"26\"\n        month_exp = \"06\"\n        cvv = \"679\"\n        id_number = \"310437017\"\n>       self.registration_page.fill_credit_card_info(credit_card, year_exp, month_exp, cvv, id_number)\n\ntest_registration.py:50: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../pages/registration_page.py:98: in fill_credit_card_info\n    self.ui_actions.frame_actions(\"fill\", self.__frame, self.__card_number, credit_card)\n../base/ui_actions.py:47: in frame_actions\n    self.wait_for_element(locator)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <base.ui_actions.UIActions object at 0x1070a8f50>\nlocator = '#card-number', timeout = 5000\n\n    def wait_for_element(self, locator, timeout=5000):\n        self.log.info(f\"Waiting for element: {locator}\")\n        try:\n            element = self.__page.locator(locator)\n            element.wait_for(state=\"visible\", timeout=timeout)\n            return element\n        except Exception as e:\n            self.log.error(f\"Element '{locator}' not found within {timeout}ms. Exception: {e}\")\n>           raise Exception(f\"Element '{locator}' not found within {timeout}ms.\") from e\nE           Exception: Element '#card-number' not found within 5000ms.\n\n../base/base_page.py:20: Exception"}, "description": "Test 07 - Fill in credit card info", "steps": [{"name": "Step 1 - Press on add credit card button", "status": "passed", "start": 1745505526690, "stop": 1745505527803}], "attachments": [{"name": "log", "source": "aba91f74-612d-46c6-891b-0e7c68d5c678-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "browser_name", "value": "'chromium'"}], "start": 1745505526690, "stop": 1745505532809, "uuid": "dc5ea34f-fbe9-42e3-ac86-2da375e3e9a6", "historyId": "2b5a2d2fc75dc6b7da2019b2473bbd8a", "testCaseId": "eca12d1be48353be73ea7caf33166409", "fullName": "tests.test_registration.TestLogin#test_07_enter_credit_card_info", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_class')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_registration"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Boriss-MacBook-Pro-4.local"}, {"name": "thread", "value": "15738-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_registration"}]}